model/event/host_event_test.go

```diff
@@ -147,6 +147,21 @@ func TestLoggingHostEvents(t *testing.T) {
 			So(eventData.Hostname, ShouldBeBlank)
 			So(eventData.TaskId, ShouldBeBlank)
 			So(eventData.TaskPid, ShouldEqual, taskPid)
+
+			// test logging multiple executions of the same task
+			err = UpdateExecutions(hostId, taskId, 0)
+			So(err, ShouldBeNil)
+
+			eventsForHost, err = Find(AllLogCollection, HostEventsInOrder(hostId))
+			So(err, ShouldBeNil)
```

Comment:
add an assertion that there should be at least one result?

Reply:
Done